src/gazebo/differential_drive_6w.cpp

```diff
@@ -48,6 +48,8 @@ namespace gazebo {
 
     constexpr auto VELOCITY_COMMAND_TOPIC = "cmd_vel";
 
```

Comment:
How did this auton sim stuff end up here?

Reply:
I pulled from master in this commit to fix the merge conflicts

Reply:
You pulled right? Not rebased? I have no idea how a commit from @ankithu got here.

Reply:
Yeah I only pulled

Reply:
Yeah this commit should probably be reverted